packages/libs/src/sdk/api/generated/full/models/AnnouncementNotification.ts

```diff
@@ -0,0 +1,109 @@
+/* tslint:disable */
+/* eslint-disable */
+// @ts-nocheck
+/**
+ * API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+import type { AnnouncementNotificationAction } from './AnnouncementNotificationAction';
+import {
+    AnnouncementNotificationActionFromJSON,
+    AnnouncementNotificationActionFromJSONTyped,
+    AnnouncementNotificationActionToJSON,
+} from './AnnouncementNotificationAction';
+
+/**
+ * 
+ * @export
+ * @interface AnnouncementNotification
+ */
+export interface AnnouncementNotification {
+    /**
+     * 
+     * @type {string}
+     * @memberof AnnouncementNotification
+     */
+    type: string;
+    /**
+     * 
+     * @type {string}
+     * @memberof AnnouncementNotification
+     */
+    groupId: string;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof AnnouncementNotification
+     */
+    isSeen: boolean;
+    /**
+     * 
+     * @type {number}
+     * @memberof AnnouncementNotification
+     */
+    seenAt?: number;
+    /**
+     * 
+     * @type {Array<AnnouncementNotificationAction>}
+     * @memberof AnnouncementNotification
+     */
+    actions: Array<AnnouncementNotificationAction>;
```

Comment:
Can you remind me what the nested `actions` array is supposed to be? Do we group notifications of a type together here?

Reply:
Yeah I think that's it https://github.com/AudiusProject/audius-protocol/blob/35ece29f0df7dabe499e30e0ccd66f3fc75a7a3b/packages/discovery-provider/src/queries/get_notifications.py#L533-L542